core/templates/components/summary-tile/subtopic-summary-tile.component.ts

```diff
@@ -0,0 +1,80 @@
+// Copyright 2015 The Oppia Authors. All Rights Reserved.
```

Comment:
The directive was created in 2015 but the new component is created this year.

Reply:
done